release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -223,10 +223,37 @@ xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.ad
 
 There are two noteworthy Ingress enhancements introduced in {product-title} 4.5:
 
-* You can enable access logs for the Ingress Controller.
-//:../networking/ingress-operator.adoc#nw-configure-ingress-access-logging_configuring-ingress[enable access logs for the Ingress Controller].
+* You can xref:../networking/ingress-operator.adoc#nw-configure-ingress-access-logging_configuring-ingress[enable access logs for the Ingress Controller].
 * You can xref:../networking/ingress-operator.adoc#using-wildcard-routes_configuring-ingress[specify a wildcard route policy through the Ingress Controller].
 
+[id="ocp-4-5-haproxy-upgraded"]
+==== HAProxy upgraded to version 2.0.14
+
+The HAProxy used for Ingress has been upgraded from version 2.0.13 to 2.0.14.
+This upgrade provides a router reload performance improvement. The router reload
+optimization is most beneficial for clusters with thousands of routes.
+
+[id="ocp-4-5-http-2-ingress-support"]
+==== HTTP/2 Ingress support
+
+{product-title} 4.5 enables transparent end-to-end HTTP/2 support in HAProxy for
+application owners to make use of its protocol capabilities, including single
+connection, header compression, binary streams, and more.
+
+To enable HTTP/2 for the connection from the client to HAProxy, a route must
+specify a custom certificate. A route that uses the default certificate cannot
+use HTTP/2. This restriction is necessary to avoid problems from connection
+coalescing, like the client re-using a connection for different routes that use
+the same certificate.
+
+The connection from HAProxy to the application Pod can use HTTP/2 only for
+re-encrypt routes and not for edge-terminated or insecure routes. This
+restriction comes from the fact that HAProxy uses Application-Level Protocol
+Negotiation (ALPN), which is a TLS extension, to negotiate the use of HTTP/2
+with the backend. This restriction also applies in {product-title} 4.4. For
```

Comment:
As we removed http/2 support in 4.4 is it necessary to mention 4.4 here? cc @Miciah

Reply:
@frobware I've removed the 4.4 reference :+1: